demos/jans-tarp/webpack.common.js

```diff
@@ -64,6 +64,22 @@ const chromeConfig = merge(commonConfig, {
                 from: path.resolve('src/static/icon.png'),
                 to: path.resolve('dist/chrome')
             },
+            {
```

Comment:
If there are common images with the same path, could you please create a shared array of objects and try to append that array in both places?

Reply:
done